pkg/agent/openflow/client.go

```diff
@@ -110,25 +114,26 @@ func (c *client) IsConnected() bool {
 	return c.bridge.IsConnected()
 }
 
-// addMissingFlows adds any flow from flows which is not currently in the flow cache. The function
-// returns immediately in case of error when adding a flow. If a flow is added successfully, it is
-// added to the flow cache. If the flow cache has not been initialized yet (i.e. there is no
-// flowCacheKey key in the cache map), we create it first.
-func (c *client) addMissingFlows(cache *flowCategoryCache, flowCacheKey string, flows []binding.Flow) error {
-	// initialize flow cache if needed
-	fCacheI, _ := cache.LoadOrStore(flowCacheKey, flowCache{})
-	fCache := fCacheI.(flowCache)
-
+// addFlows installs the flows on the OVS bridge and then add them into the flow cache. If the flow cache is existing,
+// it will return immediately, otherwise it will use Bundle to add all flows, and then add them into the flow cache.
+// If it fails to add the flows with Bundle, it will return the error and no flow cache is created.
+func (c *client) addFlows(cache *flowCategoryCache, flowCacheKey string, flows []binding.Flow) error {
+	_, ok := cache.Load(flowCacheKey)
+	// If the flow cache is existing, return immediately. Otherwise, it creates a flow cache.
```

Comment:
```suggestion
	// If a flow cache entry already exists for the key, return immediately. Otherwise, add the flows to the switch and populate the cache with them.
```